tests/Domain/Integrations/Repositories/EloquentIntegrationRepositoryTest.php

```diff
@@ -348,4 +359,27 @@ public function test_it_can_active_with_an_organization(): void
             'status' => IntegrationStatus::Active,
         ]);
     }
+
+    public function test_it_can_save_partner_status(): void
```

Comment:
No need for an extra test, this can be tested in the previous tests. The existing `assertDatabaseHas` need an extra check for `partner_status`